error.md

```diff
@@ -0,0 +1,21 @@
+## Error: with streamlit caching
```

Comment:
Given your cache change above, I believe you can remove this file. Once it's removed, happy to approve.